src/components/Alert/Alert.js

```diff
@@ -1,13 +1,13 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
-import bem from '../../utils/bem';
+import { bem } from '../../utils/bem';
```

Comment:
if we change this to named export, maybe we could also create an `index.ts` file and require it from `../utils`